tests/detect-vlan-id/test.rules

```diff
@@ -0,0 +1,10 @@
+alert ip any any -> any any (msg:"Vlan ID is equal to 200 with specific layer"; vlan.id:200,0; sid:1;)
+alert ip any any -> any any (msg:"One Vlan ID is equal to 300"; vlan.id:300; sid:2;)
+alert ip any any -> any any (msg:"Last Vlan ID is equal to 400"; vlan.id:400,-1; sid:3;)
+alert ip any any -> any any (msg:"Vlan ID is equal to 300 with specific layer"; vlan.id:0x12C,1; sid:4;)
+alert ip any any -> any any (msg:"Vlan ID at layer 1 is not equal to 200"; vlan.id:!200,1; sid:5;)
+alert ip any any -> any any (msg:"There is no VLAN ID equal to 500"; vlan.id:!500; sid:6;)
```

Comment:
Tests look good overall

But this sig is confusing `msg:"There is no VLAN ID equal to 500"` should be `vlan.id:!500, all`

Makes me think we could also add some negative tests : add signatures that do not match like `vlan.id:!300, all` (even if `vlan.id:!300` matches)

This would highlight the feature of `all`